packages/hardhat-chai-matchers/src/internal/reverted/revertedWithCustomError.ts

```diff
@@ -148,7 +150,8 @@ export function supportRevertedWithCustomError(
 function validateInput(
   obj: any,
   contract: EthersT.BaseContract,
-  expectedCustomErrorName: string
+  expectedCustomErrorName: string,
+  args: any[]
```

Comment:
I can change if you want but I don't see why, 

the 'rest parameter' pattern is used in line 36 and here it would deconstruct the array only to be retrieved as another argument inside the `validateInput` function,

Let me know what is your preferred way

Reply:
Ah, sorry, my bad. I thought this was the entry point to the assertion.